plugins/calibrecompanion.koplugin/main.lua

```diff
@@ -430,9 +541,14 @@ function CalibreCompanion:getBookCount(arg)
     local books = {
         willStream = true,
         willScan = true,
-        count = 0,
+        count = #self.book_list
     }
     self:sendJsonData('OK', books)
+    for i, _ in ipairs(self.book_list) do
+        local book = self.book_list[i]
+        book.priKey = i
+        self:sendJsonData('OK', book)
```

Comment:
Probably best to stick with our modern coding style. ;-)
```suggestion
        self:sendJsonData("OK", book)
```

Reply:
Single quotes are used across the whole file. Shall I change all of them?. See opcodes

Reply:
Sure, but maybe as a follow-up for git blame purposes in the future?

Reply:
I will use single quotes on this PR and change them later on :+1: